src/components/taskCreator/ParticipantsSelection.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import { ref, computed, watch } from 'vue';
+import { useLtcQuestStore__isTeam } from '@/stores/ltcQuestStore';
+import { teams } from '@/assets/data/teams';
+import { users } from '@/assets/data/users';
+import { useI18n } from "vue-i18n";
+
+const { t } = useI18n();
+
+const ltcQuestStore__isTeam = useLtcQuestStore__isTeam();
+
+const items = computed(() => {
+    if (ltcQuestStore__isTeam.isTeamMode) {
+        return teams.map(team => ({
+            value: team.teamId,
+            label: team.name,
+        }));
+    } else {
+        return users.map(user => ({
+            value: user.userId,
+            label: user.name,
+        }));
+    }
+});
```

Comment:
```suggestion
const items = computed(() => {
    return ltcQuestStore__isTeam.isTeamMode
        ? teams.map(team => ({ value: team.teamId, label: team.name }))
        : users.map(user => ({ value: user.userId, label: user.name }));
});
```

Jeśli coś przypisujesz warunkowo do zmiennej to używaj ternary operatora :D